plugins/modules/jira.py

```diff
@@ -59,6 +59,18 @@
       - The personal access token to log-in with.
       - Mutually exclusive with O(username) and O(password).
     version_added: 4.2.0
+  client_cert:
+    type: path
+    description:
+      - Client certificate if required.
+      - In addition to O(username) and O(password) or O(token). Not mutually exclusive.
+    version_added: 10.4.0
+  client_key:
+    type: path
+    description:
+      - Client certificate key if required.
+      - In addition to O(username) and O(password) or O(token). Not mutually exclusive.
+    version_added: 10.4.0
```

Comment:
Something that will not be solved now, let alone in this PR, is that the docs for these parameters *should* be in a doc_fragment, ansible-core has one for `url`s but that one includes a parameter .... `url`. I don't ansible-core will be changing that anytime soon (in theory it should break that in two separate fragments, one for the `url` itself and another for all these other operational parameters for HTTP transactions, but that would imply changes in a lot modules - not gonna happen).

So, it could/should be a future PR for c.g. to create doc fragments like that and use it within the collection.